native/native_main.py

```diff
@@ -16,6 +18,14 @@
 class NoConnectionError(Exception):
```

Comment:
Please bump the version :)

Reply:
[Done!][0]

[0]: https://github.com/gsbabil/tridactyl/commit/66444af20a548dba07269ed30c39246f4d6acf08#diff-0e71d767312cefb8a12ee5d2753346d1R15